src/Link Button/main.vala

```diff
@@ -0,0 +1,10 @@
+#! /usr/bin/env -S vala workbench.vala --pkg gtk4 --pkg libadwaita-1
+
+public void main () {
+    var linkbutton = (Gtk.LinkButton) workbench.builder.get_object ("linkbutton");
+    linkbutton.clicked.connect (() => message ("The link has been visited"));
```

Comment:
The signal here must be `notify["visited"]` instead of `clicked`, as in the JS demo.

Reply:
@urffsamhunt Thanks for fixing this in the Rust demo! Please make the same changes in Vala